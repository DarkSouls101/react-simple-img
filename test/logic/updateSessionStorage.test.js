import updateSessionStorage from '../../src/logic/updateSessionStorage';

describe('updateSessionStorage', () => {
  it('should update session storage', () => {
    updateSessionStorage('test');
    expect(Object.keys(JSON.parse(window.sessionStorage.getItem('__REACT_SIMPLE_IMG__')))).toMatchSnapshot();
  });
});
