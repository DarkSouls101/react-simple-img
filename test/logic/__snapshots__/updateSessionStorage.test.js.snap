// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`updateSessionStorage should update session storage 1`] = `
Array [
  "test",
]
`;
